jetty-gcloud/jetty-gcloud-session-manager/src/main/config-template/etc/sessions/gcloud/session-store.xml

```diff
@@ -44,6 +44,9 @@
               <Set name="lastNode">
                 <Property name="jetty.session.gcloud.model.lastNode" default="lastNode" />
               </Set>
+              <Set name="lastSaved">
+                <Property name="jetty.session.gcloud.model.lastSaved" default="lastSaved" />
+              </Set>
```

Comment:
What's the rationale to have some as `property` attribute and some as `Property` elements?
The code and the `*.mod` `[ini-template]` are consistent, so I guess they can all be `property` attributes?

Reply:
The `Property` element allows a default value to be set, whereas the `property` attribute does not.

Reply:
@sbordet specifically, sometime we "want" to have different defaults for embedded usage (default in the code) vs deployed usage (default in the XML).    I'm not sure we should create any more situations where the default is different depending on the run mode, but we do have several historic cases.     We did originally convert some of them to `property` elements (ie use only the default from the code), but that caused complaints.   So we need to use `Property` element whenever we want the xml to always set a value, even if the property is not set.

Reply:
I've converted all the `<Property>` elements to `property` attributes as the `defaults` are the same as the code (and the corresponding `.mod` file).